int8 ORIENTATION_LEFT_TO_RIGHT = 0
int8 ORIENTATION_RIGHT_TO_LEFT = 1

int8 orientation
---
